index.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
 var chalk = require('chalk');
+var isSupported = process.platform !== 'win32' || typeof document !== 'undefined' && document.nodeType === 9;
```

Comment:
`typeof document !== 'undefined' && document.nodeType === 9` should be defined above in a separate variable first called `isBrowser`.